js/app.js

```diff
@@ -117,17 +144,54 @@ $("#mc-embedded-subscribe-form").ajaxForm({
 
       return;
     }
-    var amount = $('input[name="AMOUNT"]:checked').val();
-    $('#amount-to-donate').text('$' + amount);
-    $('#mc-embedded-subscribe-form').addClass('hide');
-    $('#confirmation').removeClass('hide').hide().fadeIn();
-    window.scrollTo(0, $('#confirmation').offset().top - 30);
+  }
 
-    ga('send', 'event', {
-      eventCategory: 'subscription',
-      eventAction: 'success',
-      eventLabel: 'join flow',
-      nonInteraction: true,
-    });
+  var amount = $('input[name="AMOUNT"]:checked').val();
+  $('#mc-submit-button').text("Loading...");
+
+  ga('send', 'event', {
+    eventCategory: 'subscription',
+    eventAction: 'success',
+    eventLabel: 'join flow',
+    nonInteraction: true,
+  });
+
+  paypalAction(amount);
+}
+
+// Pre-fill form values
+function getUrlParameter(sParam) {
+  var sPageURL = decodeURIComponent(window.location.search.substring(1)),
+    sURLVariables = sPageURL.split('&'),
+    sParameterName,
+    i;
+
+  for (i = 0; i < sURLVariables.length; i++) {
+    sParameterName = sURLVariables[i].split('=');
+
+    if (sParameterName[0] === sParam) {
+      return sParameterName[1] === undefined ? true : sParameterName[1];
```

Comment:
Had to look this one up!

`sParameterName[1]` resolves to `''`, which in turn is interpreted as `false` in the assignment below.

http://stackoverflow.com/a/8693015

Confirmed via QA.